packages/@aws-cdk/aws-ec2/lib/vpc-ref.ts

```diff
@@ -335,3 +361,34 @@ export interface VpcSubnetImportProps {
    */
   subnetId: string;
 }
+
+/**
+ * If the placement strategy is completely "default", reify the defaults so
+ * consuming code doesn't have to reimplement the same analysis every time.
+ *
+ * Returns "private subnets" by default.
+ */
+function reifyDefaultsPlacement(placement: VpcPlacementStrategy): VpcPlacementStrategy {
+    if (placement.subnetsToUse !== undefined && placement.subnetName !== undefined) {
+      throw new Error('At most one of subnetsToUse and subnetName can be supplied');
```

Comment:
Would phrase this as `Only one of subnetsToUse and subnetName can be supplied`. The "at most" to me is implied by "can". Not feeling very strongly though.

Reply:
You're right on phrasing.